src/main/bash/sdkman-init.sh

```diff
@@ -156,6 +156,8 @@ if [[ -z "${sdkman_curl_retry_max_time}" ]]; then sdkman_curl_retry_max_time=60;
 # set curl to continue downloading automatically
 if [[ -z "${sdkman_curl_continue}" ]]; then sdkman_curl_continue=true; fi
 
+if [[ -z "$sdkman_selfupdate_feature" ]]; then sdkman_selfupdate_feature=true; fi
```

Comment:
This should never happen if the config is added by the sdkman-hooks service in `install.sh` and `update.sh`. I think we can remove it.

Reply:
I thought about not exposing the flag to non-Homebrew users, but I can see that it makes sense to add it to the sdkman-hooks service.

Reply:
Perhaps you can add a comment before to say why you've got this line here.